docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,86 @@
+* Improve performance for [file compression and decompression](file-compression)
+  of ORC and Parquet files used for [exchange spooling with fault-tolerant
+  execution mode](fte-exchange-manager). ({issue}`22532`)
+* Extend the Kafka event listener to send split completion events. ({issue}`23065`)
+
+## JDBC driver
+
+* Publishing a JDBC driver JAR without bundled, third-party dependencies. ({issue}`22098`)
+
+## BigQuery connector
+
+* Fix failures with queries using table functions when `parent-project-id` is
+  defined. ({issue}`23041`)
+
+## Blackhole connector
+
+* Add support for replacing tables. ({issue}`23004`)
+
+## Delta Lake connector
+
+* Add support for creating tables with
+  [deletion vector](https://docs.delta.io/latest/delta-deletion-vectors.html).
+  ({issue}`22104`)
+* Improve performance for concurrent write operations on S3 by using lock-less
+  Delta Lake write reconciliation. ({issue}`23145`)
+* Improve performance for [file compression and decompression](file-compression)
+  of Parquet files. ({issue}`22532`)
+* Fix SSE configuration when using S3SecurityMapping with kmsKeyId configured. ({issue}`23299`)
+
+## Hive connector
+
+* Improve performance of queries that scan a large number of partitions. ({issue}`23194`)
+* Improve performance for [file compression and decompression](file-compression)
+  of ORC and Parquet files. ({issue}`22532`)
+
+## Hudi connector
+
+* Improve performance for [file compression and decompression](file-compression)
+  of Parquet files. ({issue}`22532`)
+
+## Iceberg connector
+
+* Improve performance for [file compression and decompression](file-compression)
+  of ORC and Parquet files. ({issue}`22532`)
+
+## Memory connector
+
+* Add support for renaming schemas. ({issue}`22659`)
+
+## Prometheus connector
+
+* Fix reading large Prometheus responses. ({issue}`23025`)
+
+## SPI
+
+* Remove the deprecated `ConnectorMetadata.createView` method. ({issue}`23208`)
+* Remove the deprecated `ConnectorMetadata.beginRefreshMaterializedView` method
+  without the `RefreshType` argument. ({issue}`23212`)
+* Remove the deprecated `ConnectorMetadata.finishInsert` method without the
+  `sourceTableHandles` argument. ({issue}`23213`)
+* Remove the deprecated `ConnectorMetadata.createTable(ConnectorSession session,
+  ConnectorTableMetadata tableMetadata, boolean ignoreExisting)` method.
+  ({issue}`23209`)
+* Remove the deprecated `ConnectorMetadata.createTable(ConnectorSession session,
+  ConnectorTableMetadata tableMetadata, boolean ignoreExisting)` method.
+  ({issue}`23209`)
+* Remove the deprecated `ConnectorMetadata.beginCreateTable` method without the
+  `replace` argument. ({issue}`23211`)
+* Remove the deprecated `ConnectorSplit.getInfo method` method. Use the
+  `getSplitInfo` method instead. ({issue}`23271`)
+* Remove the deprecated `DecimalConversions.realToShortDecimal` method. Use the
+  `realToShortDecimal(float value, long precision, long scale)` method instead.
+  ({issue}`23275`)
```

Comment:
this reads weirdly since we say replace method with same method name with no idea how the old method's signature looked like.

Reply:
Fixed.